core/contracts/financial-templates/implementation/FeePayer.sol

```diff
@@ -63,9 +63,9 @@ abstract contract FeePayer is Testable {
      * @notice Constructs the FeePayer contract. Called by child contracts.
      * @param collateralAddress ERC20 token that is used as the underlying collateral for the synthetic.
      * @param finderAddress UMA protocol Finder used to discover other protocol contracts.
-     * @param isTest whether this contract is being constructed for the purpose of running tests.
+     * @param timerAddress Contract that stores the current time in a testing environment. Should be set to 0x0 for production environments that use live time.
```

Comment:
```suggestion
     * @param timerAddress Contract that stores the current time in a testing environment. 
     * Should be set to 0x0 for production environments that use live time.
```

Reply:
This string is way too long. Can we split them and try keep it below the 120 char limit?

Reply:
Yup I'll make this change throughout the PR